backend/globaleaks/db/migrations/update_31/__init__.py

```diff
@@ -182,6 +189,9 @@ def migrate_Node(self):
                 new_node.contexts_clarification = old_node.context_selector_label
                 continue
 
+            if v.name == 'whistleblowing_receipt_prompt':
+                continue
```

Comment:
it is in the appdata dictionary that is freshly loaded from the appdada.json file!